src/System-BasicCommandLineHandler.package/BasicCommandLineHandler.class/instance/activate.st

```diff
@@ -1,8 +1,10 @@
 activation
 activate
 	
-	[ self handleArgument: (self arguments 
+	[ [ self handleArgument: (self arguments 
 			ifEmpty: [ '' ] 
 			ifNotEmpty: [ :arguments| arguments first ])] 
 		on: Exit 
-		do: [ :exit | ^ self handleExit: exit ]
\ No newline at end of file
+		do: [ :exit | ^ self handleExit: exit ] ]
+			forkAt: Processor userSchedulingPriority
```

Comment:
Not needed, but it's better because it will make startup actions run with a normal priority instead of maximum priority. This will avoid problems and simplify the case of a command line handler trying to save an image: otherwise the command line handler will run in max priority and we will have to be very careful all the time to synchronize processes in max priority.